web/src/lib/stores/assets-store.svelte.ts

```diff
@@ -267,7 +267,7 @@ export class AssetBucket {
 
     const year = utcDate.get('year');
     const month = utcDate.get('month');
-    const bucketDateFormatted = utcDate.toJSDate().toLocaleString(get(locale), {
+    const bucketDateFormatted = utcDate.toJSDate().toLocaleString(get(locale) || 'en-US', {
```

Comment:
Do you need to default the language to `en-US` everywhere that you get the locale and not just here? If so, it's probably better to do it inside the store so that you can do it in a single place and it applies everywhere and you don't have to update all the individual locations. The cleanest way to do that would probably be to refactor it from a store to a Svelte 5 class

Reply:
I agree this should probably be done where the store is created

Reply:
Done